pulp_rpm/app/tasks/synchronizing.py

```diff
@@ -58,8 +58,8 @@
     PackageCategory,
     PackageEnvironment,
     PackageLangpacks,
-    RpmRemote,
     RpmRepository,
+    RpmRemote,
```

Comment:
Why did it move one line below?

Reply:
due to refactor I put it back to wrong line.